inc/sdk/system.h

```diff
@@ -0,0 +1,14 @@
+#ifndef GBSDK_SYSTEM_H
+#define GBSDK_SYSTEM_H
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#define CPU_DMG 0x01 // Dot Matrix Game
+#define CPU_CGB 0x11 // Color GameBoy
+#define CPU_MGB 0xFF // Mini GameBoy (Pocket GameBoy)
```

Comment:
0xFF is also an SGB2.

Reply:
Right, I can add that. I was just copying that text from hardware,inc.